src/main/java/com/example/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -34,6 +36,11 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ExceptionDto handleRegistrationException(RegistrationException ex) {
```

Comment:
Let's return `ResponseEntity<Object>` as in handleMethodArgumentNotValid method. Be consistent